iso_config.yaml

```diff
@@ -0,0 +1,37 @@
+log_file: "/var/tmp/host-os-iso-creator.log"
+mock_config: "./extras/centOS/7.2/mock/spin-iso-epel-7-ppc64le.cfg"
+rpm_dir: "./results"
+kickstart_file: "host-os.ks"
+hostos_package_group: "virtualization-hypervisor"
+package_group_list:
```

Comment:
I did some tests and was able to reduce the package list to a minimum set, which speeds up the process and reduces the ISO size, hence, I will keep your suggestion and leave a tiny set of packages.